packages/react-scripts/config/webpack.config.js

```diff
@@ -501,7 +554,10 @@ module.exports = function(webpackEnv) {
             // of CSS.
             // By default we support CSS Modules with the extension .module.css
             {
-              test: cssRegex,
+              test: {
```

Comment:
Hmm part of me thinks this is a good opportunity to remove the horrid logic around whether to use cssmodules or now ...

Reply:
Not sure what you mean here?

Reply:
`cssModulesEnabled` - is still used so for now will have to use it unless we decide to remove it in this major version and force consumers to update?